README.md

```diff
@@ -8,7 +8,13 @@ This page and repository are maintained by [JSK](http://www.jsk.t.u-tokyo.ac.jp
 
 EusLisp is an integrated programming system for the research on intelligent robots based on Common Lisp and Object-Oriented programming. The principal subjects in the field of robotics research are sensory data processing, visual environment recognition, collision avoiding motion planning, and task planning. In either problem, three dimensional shape models of robots and environment play crucial roles. A motivation to the development of EusLisp was a demand for an extensible solid modeler that can easily be made use of from higher level symbolic processing system. Investigations into traditional solid modelers proved that the vital requirement for their implementation language was the list processing capability to represent and manage topology among model components. Numerical computation power was also important, but locality of geometric computation suggested the provision of vector/matrix functions as built-ins would greatly ease programming. (From [EusLisp Manual, Part I, EusLisp Basics, 1. Introduction](http://staff.aist.go.jp/t.matsui/eus/intro/eusintro.html ))
 
-### How To Download and Compile
+### Installation
+
+#### Binary Installation (Recommended)
+
+Please check http://github.com/euslisp/jskeus for easy binary installation and how to run current demo programs. Most of programs in euslisp/eus/lib/ euslisp/eus/contrib/ directories are out-dated.
```

Comment:
Wouldn't it be EusLisp/lib/ and EusLisp/contrib/ instead of euslisp/eus/lib/ and euslisp/eus/contrib/? I cannot find the latter ones, neither on the EusLisp repository nor on the jskeus one.

Reply:
Or rather, add the note that "The installation directory is assumed to be "/usr/local/eus/" and should be set to the global variable *eusdir*", as noted on the installation manual (so that that phrase would make more immediate sense).

Reply:
I made those changes and pushed already. They can be seen on the original PR: https://github.com/euslisp/EusLisp/pull/276/files